tox.ini

```diff
@@ -0,0 +1,17 @@
+[tox]
+envlist = py26,py27,py34,py35,py36
+
+[testenv]
+commands = nosetests --nologcapture
+basepython =
+	py26: python2.6
+	py27: python2.7
+	py34: python3.4
+	py35: python3.5
+	py36: python3.6
```

Comment:
Specifying `basepython` this way is optional in tox (as this is the default setting) [according to the documentation](http://tox.readthedocs.io/en/latest/example/general.html?highlight=basepython#basepython-defaults-overriding).